Makefile

```diff
@@ -1,38 +1,44 @@
 PYJ_GLOBALS='$$,chrome,Marka,alertify'
+OUTPUT_DIR=output
+SOURCE_DIR=src
+VPATH = $(SOURCE_DIR)
 
 %.js: %.pyj
 	rapydscript lint --globals $(PYJ_GLOBALS) $(PYJ_FLAGS) $<
-	rapydscript $(PYJ_FLAGS) $< --output $@
-	sed -e ':a;N;$$!ba;s/async;\n/async/g' -i $@
+	rapydscript $(PYJ_FLAGS) $< --output $(OUTPUT_DIR)/$@
+	sed -e ':a;N;$$!ba;s/async;\n/async/g' -i $(OUTPUT_DIR)/$@

```

Comment:
Since `src` is on the VPATH, Make discovers the .pyj dependencies directly instead of using the full path to them.
So the output for this goes to `output/*.js`